packages/app/features/Send/TransactionConfirmation/components/RecipientInfo.tsx

```diff
@@ -24,13 +24,15 @@ export const RecipientInfo: FC<Props> = () => {
 			(type === 'Token' ? token?.network : nftCollectible?.network),
 	);
 
+	const tokenForFeeName = tokenForFee?.metadata?.symbol ?? 'Unknown';
```

Comment:
check this

Reply:
i handle this case for some tokens which are mint unofficially so that we cannot fetch their metadata on solana